adversarial/advbox/attacks/gradientsign.py

```diff
@@ -36,3 +36,39 @@ def _apply(self, image_label, epsilons=1000):
 
 
 FGSM = GradientSignAttack
+
+
+class IteratorGradientSignAttack(Attack):
+    """
+    This attack was originally implemented by Alexey Kurakin(Google Brain).
+    Paper link: https://arxiv.org/pdf/1607.02533.pdf
+    """
+
+    def _apply(self, image_label, epsilons=100, steps=10):
```

Comment:
I'm curious about what's the label here. The true label? Or some fake label we made?

Reply:
image_label is the same as GradientSignAttack.
image_label(list): The image and label tuple list of one element. 
The label is source label（non－target).
Target attack need new interface.

Reply:
Got it. Thanks!